docroot/themes/custom/twentyeighteen/sass/components/403-404/_403-404.scss

```diff
@@ -0,0 +1,51 @@
+// 403 and 404 Page Styling
+//
+// Styling 403 and 404 pages
+//
+// Markup: 403-404.twig
+//
+// Style guide: components.403-404
+
+.error-container {
+  position: relative;
+}
+
+.body-sidebars-none .error-container section#content .error-notice {
+  padding: 0;
+}
+
+.error-container {
+  padding-bottom: 10em;
+  background-color: rgba(221, 198, 102, 1);
+
+  @include breakpoint($small-max) {
+    padding-bottom: 0;
+  }
+}
+
+.notfound-backing {
+  background-color: rgba(52, 109, 117, 1);
+}
+
+.error-container #content-area {
```

Comment:
This can be changed to `.error-notice` instead of this once those changes in the twig files are in place.